extra/gcp/external-worker-gcp-template.jinja

```diff
@@ -1,6 +1,6 @@
 ---
 
-{% set name_prefix = properties["customerTag"] + "-" + properties["projectTag"] + "-" + properties["environmentTag"] %}
+{% set name_prefix = properties["projectTag"] + "-" + properties["roleTag"] + "-" + properties["environmentTag"] %}
```

Comment:
Not sure to get why we changed it ?

Reply:
It was mainly for consistency with the AWS deployment that is not using the customer name in naming.